### DISCUSSION TITLE
Styles, '5' buttons, Generate and Enqueue buttons moved

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#1514] [General]
Whyyyyy?

![image](https://github.com/vladmandic/automatic/assets/1969381/c5e69dd9-b827-4e1b-acb3-6540a3c9e517)

**Comment by vladmandic** [comment]
not on my system

![image](https://github.com/vladmandic/automatic/assets/57876960/cee3f7d8-c188-4f0c-a765-7567b2a76ced)

**Reply by brknsoul** [reply]
OK, must be an errant extension.. time to go back into the weeds!

**Reply by brknsoul** [reply]
Ok, it seems that Agent Scheduler is screwing up for some reason;
If I use the setting to move the Enqueue button, it looks normal;
![image](https://github.com/vladmandic/automatic/assets/1969381/b4edacbd-3677-47b9-b609-9cd7cdd43882)

Created https://github.com/ArtVentureX/sd-webui-agent-scheduler/issues/50
